src/app/dataset/datasetform/datasetform.component.html

```diff
@@ -177,7 +177,7 @@
         <label class="dataset-label" for="notes">{{ 'notes' | translate }}</label>
         <textarea id="notes" formControlName="notes" (blur)="saveTempData()"></textarea>
       </div>
-      <div class="form-group">
+      <div class="form-group" *ngIf="!isNew">
```

Comment:
This is good, but the frontend still sends the default "not fit for publication" to the backend (see `datasetform.component.ts` line 242). Perhaps we can send another default (i.e. `PublicationFitness.FIT`)?